diff --git a/build/all.gyp b/build/all.gyp
index 2813ca2..01a9a22 100644
--- a/build/all.gyp
+++ b/build/all.gyp
@@ -276,6 +276,13 @@
       ],
     }, # target_name: All
     {
+      'target_name': 'sprocket',
+      'type': 'none',
+      'dependencies': [
+        '../sprocket/sprocket.gyp:sprocket',
+      ],
+    }, # target_name: sprocket
+    {
       'target_name': 'All_syzygy',
       'type': 'none',
       'conditions': [
