diff --git a/BUILD.gn b/BUILD.gn
index 23e67d9..025fc95 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -781,6 +781,7 @@ group("gn_only") {
       "//third_party/webrtc/system_wrappers:metrics_default",
       "//ui/display/types",
       "//ui/shell_dialogs:shell_dialogs_unittests",
+      "//sprocket:sprocket",
     ]
 
     if (target_cpu == "x86" || target_cpu == "x64") {
@@ -818,6 +819,7 @@ group("gn_only") {
     deps += [
       "//build/android/gyp/test:hello_world",
       "//build/android/incremental_install:bootstrap_java",
+      "//sprocket/android:sprocket_apk",
     ]
   }
 
