diff --git a/chrome/test/chromedriver/chrome/device_manager.cc b/chrome/test/chromedriver/chrome/device_manager.cc
index 8e77f0c..48ec7e1 100644
--- a/chrome/test/chromedriver/chrome/device_manager.cc
+++ b/chrome/test/chromedriver/chrome/device_manager.cc
@@ -55,6 +55,12 @@ Status Device::SetUp(const std::string& package,
     device_socket = "content_shell_devtools_remote";
     command_line_file = "/data/local/tmp/content-shell-command-line";
     exec_name = "content_shell";
+  } else if (package.compare("hu.uszeged.sprocket_apk") == 0) {
+    // Sprocket.
+    known_activity = ".SprocketActivity";
+    device_socket = "sprocket_devtools_remote";
+    command_line_file = "/data/local/tmp/sprocket-command-line";
+    exec_name = "sprocket";
   } else if (package.compare("org.chromium.chrome.shell") == 0) {
     // ChromeShell
     known_activity = ".ChromeShellActivity";
