diff --git a/tools/gritsettings/resource_ids b/tools/gritsettings/resource_ids
index c1f41e0..308b08e 100644
--- a/tools/gritsettings/resource_ids
+++ b/tools/gritsettings/resource_ids
@@ -182,6 +182,9 @@
   "content/shell/shell_resources.grd": {
     "includes": [24650],
   },
+  "sprocket/sprocket_resources.grd": {
+    "includes": [25600],
+  },
   # This file is generated during the build.
   "<(SHARED_INTERMEDIATE_DIR)/content/browser/tracing/tracing_resources.grd": {
     "includes": [24900],
